socs/Lakeshore/Lakeshore372.py

```diff
@@ -1295,7 +1295,41 @@ def set_curve(self, _file):
             self._set_data_point(point[0], point[1], point[2])
 
         # refresh curve attributes
-        return self.get_header()
+        self.get_header()
+
+    def _check_curve(self, _file):
+        """After setting a data point for calibration curve,
+        use CRVPT? command from get_data_point() to check
+        that all points of calibration curve  were uploaded.
+        If not, re-upload points.
+
+        :param _file: calibration curve file
+        :type _file: str
+        """
+
+        with open(_file) as f:
+            content = f.readlines()
+
+        #skipping header info
+
```

Comment:
Similar to the below comment about blank lines, I'd remove this extra line too so that the "skipping header info" comment, which I believe corresponds to this block, is directly on top of it.